src/OnePlusBot/Base/Core.cs

```diff
@@ -29,6 +29,20 @@ public async Task MainAsync()
             _bot.ReactionAdded += OnReactionAdded;
 
             _bot.ReactionRemoved += OnReactionRemoved;
+            
+            try
```

Comment:
Check if it doesn't exist before trying to read, avoiding the try catch.
Create the file if it doesn't exist and continue workflow normally.